src/odemis/model/_core.py

```diff
@@ -19,6 +19,8 @@
 You should have received a copy of the GNU General Public License along with
 Odemis. If not, see http://www.gnu.org/licenses/.
 '''
+import tempfile
+
 from past.builtins import basestring
 import Pyro4
```

Comment:
If you want you could add another commit to update the import order according to the new coding guidelines and replace the `'''` with `"""` on line 2 and line 21